packages/shortest/src/ai/tools/index.ts

```diff
@@ -30,6 +30,42 @@ export const AITools = [
       required: ["action", "username", "password"],
     },
   },
+  {
+    name: "check_email",
```

Comment:
What about 

```suggestion
    name: "view_email",
```

Reply:
I prefer check (this is why the same PR would be good)